doc/faq.rst

```diff
@@ -49,9 +49,9 @@ looks like this:
     Fatal: repository contains errors
```

Comment:
The error message here is also outdated. The current variant is:
```
using temporary cache in /tmp/restic-check-cache-168381449
repository 86923eb5 opened successfully, password is correct
created new cache in /tmp/restic-check-cache-168381449
create exclusive lock for repository
load indexes
check all packs
pack 8f5c55d37d28a86cb9a20adb030d115f07bbecf05f3506998d0254f2b39b2496: not referenced in any index
pack c02e2919d799891c5ca752af8829ca1401b0a0cf135ed8b996d0e7929f8a6455: not referenced in any index
pack a9047d2ccabf343754a0108e49aba885bbe4c18277ee38b757ab7764aa721fb0: not referenced in any index
pack eeb0794fab182cbac3a1fa5bd61446782009445f98f362eb8185e587f4257125: not referenced in any index
pack bf34a6d44cceff21c29c4da44cf10245b7aa9ed1698551ce915d5d687d10c057: not referenced in any index
pack a07d4572848962f46269628cb32b1f8008d384b0028b9a64cc4487bf71dca853: not referenced in any index
pack d40ad98368d52b75b785abd8c7dc6642428f8e1815c21486934a34f9c5729f88: not referenced in any index
7 additional files were found in the repo, which likely contain duplicate data.
You can run `restic prune` to correct this.
check snapshots, trees and blobs
[0:00] 100.00%  16 / 16 snapshots
no errors were found
```

My suggestion would be to cut that down to
```
create exclusive lock for repository
load indexes
check all packs
pack 819a9a52e4f51230afa89aefbf90df37fb70996337ae57e6f7a822959206a85e: not referenced in any index
pack de299e69fb075354a3775b6b045d152387201f1cdc229c31d1caa34c3b340141: not referenced in any index
2 additional files were found in the repo, which likely contain duplicate data.
You can run `restic prune` to correct this.
check snapshots, trees and blobs
[0:00] 100.00%  16 / 16 snapshots
no errors were found
```

Reply:
I updated my branch with your proposal.

However, the current variant in `master` is already a bit different to the current variant you describe above...?

Reply:
I'm not sure which reference you're referring to? The first log snippet is the output I get when using the latest master branch. The difference to my suggestion is that it removes the first three lines, which only add noise to the example.

Reply:
Ah ok, then I misunderstood. I thought you where referring to the current text in `faq.rst`.

However, I took the contents of the second snippet and inserted it into `faq.rst` in my branch. That should be what you suggested, isn't it?

Reply:
It is. I was just confused by the reference to the current variant.